Release 2.27.0 (#1045)
luxonis/depthai-core: README.md

MacOS: `brew install pcl`
Linux: `sudo apt install libpcl-dev`

> > ℹ️ On Linux distributions based on RPMs, you need to install `perl-core` for us to OpenSSL dependency.

Author: `perl-core` is a build time dependency for openssl, not runtime.

> > ℹ️ On Linux distributions based on RPMs, you need to install `perl-core` for us to OpenSSL dependency.

Reviewer: Ah, got it. Thanks!

Then the suggested change should be:
```suggestion
> ℹ️ On Linux distributions based on RPMs, you need to install `perl-core` required by OpenSSL dependency.
```
